.github/workflows/release.yml

```diff
@@ -36,7 +36,9 @@ jobs:
       - run: git checkout release
       - run: git push --set-upstream origin release
       - run: python -m bumpver update --${{ inputs.release_type }}
-      - run: eval $(bumpver show --env) # Saving the new version in an environment variable
-      - run: gh pr create --title "Release $CURRENT_VERSION" --body "Release $CURRENT_VERSION" --base main --head release --repo ${{ github.repository }} --reviewer SRFU-NN
-      - run: gh pr merge --rebase --delete-branch --auto
-      - run: gh release create $CURRENT_VERSION --title $CURRENT_VERSION --notes "Release $CURRENT_VERSION" --verify-tag
+      - run: |
+          eval $(python -m bumpver show --env) # Saving the new version in an environment variable
+          GIT_HASH=$(git rev-parse HEAD) # Saving the git hash in an environment variable
+          gh pr create --title "Release $CURRENT_VERSION" --body "Release $CURRENT_VERSION" --base main --head release --repo ${{ github.repository }} --reviewer SRFU-NN
+          gh pr merge --rebase --delete-branch --auto
```

Comment:
LGTM, but just a question for my learning: why are we rebasing, when merging? (and is the incomming branch or 'main' rebased?)

Reply:
We do that because I thought I was smart, but it turns out I wasn't (in this case 😉).

I thought it would mean that the commit tree was linear, with the tag on `origin/main`. But since the rebase creates new releases, it is dangling off the main tree, and we have two commits with identical commit message. The code in the two commits is identical, so it isn't a huge problem, but it isn't very nice. We should probably just go back to normal merging

After rebase, the commit tree looks like this:
```
* XXX (origin/main, origin/HEAD) Bump version 0.9.2 -> 0.9.3
| * YYY (tag: 0.9.3, origin/release) Bump version 0.9.2 -> 0.9.3 
|/ 
* ZZZ Reviewed commit
```

With merge, it would look like this:
```
* XXX (origin/main, origin/HEAD) Merging
|\
| * YYY (tag: 0.9.3, origin/release) Bump version 0.9.2 -> 0.9.3 
|/ 
* ZZZ Reviewed commit
```

And to answer you last question: Every commit in the incoming branch is rebased onto the `main` branch.